shared/common-adapters/text.desktop.js

```diff
@@ -69,10 +63,10 @@ class Text extends Component<void, Props, void> {
 
     return <span
       title={this.props.title}
-      ref={this._setRef}
+      ref={this.props.allowHighlightText ? this._setRef : undefined}
```

Comment:
the only reason we need a ref is if you highlight. instead of having to pay for that we opt into it (used 2 times in the code)